ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -286,7 +286,16 @@ - (void)_rebuildBridge
       [self _enforceDesiredDeviceOrientation];
       [self _invalidateRecoveryTimer];
       [[EXKernel sharedInstance] logAnalyticsEvent:@"LOAD_EXPERIENCE" forAppRecord:self.appRecord];
-      [self.appRecord.appManager rebuildBridge];
+
+      @try {
```

Comment:
wrapping this seems fairly broad. I'm not too familiar with this part of the codebase but it's usually preferable to narrow the scope of the try/catch unless we're trying to set up a general purpose error boundary, in which case we may want this try/catch even higher.

I guess what I'm saying is: if we're not trying to create a general purpose high-level error boundary, is there a narrower scope that we can try/catch to just isolate manifest exceptions? As it's written this would capture a lot of potential thrown errors, some of which could be things like syntax errors, etc that we do want to crash the app (I think).

Reply:
Normally I'd agree, but here we forward the error to a custom error handler rather than throwing it for the device to handle (which it does by crashing). The only difference between this and what we currently do, is now you don't have to rebuild from source to determine what went wrong.